src/java.base/share/classes/sun/security/ssl/ServerHello.java

```diff
@@ -745,10 +752,51 @@ private static CipherSuite chooseCipherSuite(
             }
 
             // no cipher suites in common
+            if (SSLLogger.isOn && SSLLogger.isOn("ssl,handshake")) {
+                printServerSocketConfig(shc, null);
+            }
```

Comment:
Similarly, I may break down the debug log closer to the actions.

Reply:
See my previous reply.